js/components/controlBar.js

```diff
@@ -48,6 +46,28 @@ var ControlBar = React.createClass({
     window.removeEventListener('orientationchange', this.closePopovers);
   },
 
+  /**
+   * Restores the focus of a previously selected control bar item.
+   * This is needed as a workaround because switching between play and pause states
+   * currently causes the control bar to re-render.
```

Comment:
@public or @private

Reply:
Since this is a component I don't think you could call it externally